misc/pytorch_toolkit/chest_xray_screening/tests/test_train.py

```diff
@@ -35,21 +35,15 @@ def setUpClass(cls):
                 image_path, transform=True)
             cls.data_loader_train = DataLoader(
                 dataset=dataset_train,
-                batch_size=2,
```

Comment:
why do you update it?

Reply:
otherwise train tests for this model fail on the CI

Reply:
what is the value now?